api/filters.go

```diff
@@ -492,7 +493,15 @@ func (api *FilterAPI) checkFilterOptions(ctx context.Context, newFilter *models.
 
 func (api *FilterAPI) getVersion(ctx context.Context, dataset *models.Dataset) (*datasetAPI.Version, error) {
 
-	dimensions, err := api.datasetAPI.GetVersion(ctx, dataset.ID, dataset.Edition, strconv.Itoa(dataset.Version))
+	dimensions, err := api.datasetAPI.GetVersion(ctx,
+		"",
```

Comment:
Should we not be passing user auth token in here? If the request is from a user then we should be using the user Auth token to validate if user can retrieve this information?

I think getVersion function signature should contain the user auth token and possibly collection id too so that it can be used when calling `api.datasetAPI.GetVersion(...` func if it exists in `r.Context()`

Reply:
Ah just seen you do exactly this on line 543 and 545, so why not here?

Reply:
This was a tricky part of the changes - It wasn't clear when I needed it and when I didn't. I have aimed to replicate exactly what was happening before. I will double check that the changes reflect the previous behaviour.